spec.bs

```diff
@@ -456,6 +456,7 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
         - |document| is not [=allowed to use=] the <code><a href="#browsing-topics-policy-controlled-feature">browsing-topics</a></code> feature.
         - |document| is not [=allowed to use=] the <code><a href="#interest-cohort-policy-controlled-feature">interest-cohort</a></code> feature.
         - The user preference setting disallows the access to topics from |topLevelDocument| given |document|'s [=Document/origin=].
+        - Access to topics from |topLevelDocument| given |document|'s [=Document/origin=] is disabled due to some other user agent-defined mechanism, like lack of <a href="https://github.com/privacysandbox/attestation">enrollment</a>.
```

Comment:
Besides the document.browsingTopics() API, the headers approach also does the settings check (i.e. Step 17 of [15.8. Append or modify a request Sec-Browsing-Topics header](https://patcg-individual-drafts.github.io/topics/#append-or-modify-a-request-sec-browsing-topics-header-header)). You should also add the sentence there.

Optional: since there are two places referencing "enrollment", you can add the link to `<pre class="anchors">` section at the top, and reference it via `[=enrollment=]` in the main text here.

```
<pre class="anchors">

...
...

html; urlPrefix: https://github.com/privacysandbox/attestation
    type: dfn
        text: enrollment
</pre>
```